notify/api/v1/api.py

```diff
@@ -15,21 +15,44 @@
 
 import logging
 
+import jsonschema
 import flask
 
 from notify import config
-
+from notify.drivers.salesforce import salesforce
 
 LOG = logging.getLogger("api")
 LOG.setLevel(config.get_config().get("logging", {}).get("level", "INFO"))
 
+notify_backends_config = config.get_config()['notify_backends']
+sfdc_config = notify_backends_config['sf']['salesforce']['properties']
+
+
+sfdc_oauth2 = salesforce.OAuth2(
```

Comment:
There is rule in python, you shouldn't initialize anything on level module. 

This won't work well if add more then one driver and have specified in config only one

so you should basically here create one global variable and method here

```
SFDC = None 

def get_sfdc(self, )
    global SFDC
    if not SFDC: 
        do auth
```